src/TestUtils/src/Microsoft.Maui.IntegrationTests/SimpleTemplateTest.cs

```diff
@@ -11,11 +11,11 @@ public class SimpleTemplateTest : BaseTemplateTests
 	[TestCase("maui", DotNetPrevious, "Release", false, "", "")]
 	[TestCase("maui", DotNetCurrent, "Debug", false, "", "")]
 	[TestCase("maui", DotNetCurrent, "Release", false, "", "TrimMode=partial")]
-	[TestCase("maui", DotNetCurrent, "Debug", false, "--sample-content", "")]
-	[TestCase("maui", DotNetCurrent, "Release", false, "--sample-content", "TrimMode=partial")]
+	//[TestCase("maui", DotNetCurrent, "Debug", false, "--sample-content", "")]
```

Comment:
Could create an issue and here include a comment like:
`// TODO: Re-enable tests once the Community Toolkit supports .NET 10. More details: https://github.com/dotnet/maui/issues/99999`?

Reply:
Done